app/services/event_builder.rb

```diff
@@ -13,8 +13,10 @@ def find_pull_request
     Events::PullRequest.find_by!(github_id: @payload['pull_request']['id'])
   end
 
-  def find_first_review_request(pull_request, reviewer_id)
-    pull_request.review_requests.where(reviewer_id: reviewer_id).order(:created_at).first
+  def find_last_review_request(pull_request, reviewer_id)
+    pull_request.review_requests.where(reviewer_id: reviewer_id).last
+  rescue StandardError
```

Comment:
Can we be more specific here? What exception are we expecting if there is no review_request?